schema/2.0.0/seurat_encoding.md

```diff
@@ -220,7 +227,13 @@ Each available embedding is stored as a named element of the slot `reductions`.
     </tr>
     <tr>
       <th>Value</th>
-        <td><code>DimenReduc</code> object</td>
+        <td><code> SeuratObject::DimenReduc</code> object</td>
     </tr>
 </tbody></table>
 <br>
+
+The matrix with reductions is stored in the slot `cell.embeddings` of the `SeuratObject::DimenReduc`. Rows correspond to cells and are named with cell IDs; columns correspond to reductions and are named `[key]_[number]`, where `key` is an alphanumeric string and `number` is an integer, e.g. `PC_1`.
+
+### Dataset Metadata
+
+At the moement a Seurat object downloaded from cellxgene does not contain dataset metadata.
```

Comment:
moement -> moment

Should there be a reference to `uns` ?

Would it be helpful to have a cellxgene seurat encoding version instead of ("At the moment")?

Reply:
I think having an encoding version will help. Should we start at 2.0.0 to match it with the schema?

Reply:
To be more precise, that would probably need to be included in the seurat object for easy introspection of downloads. And we're back to the `uns` problem.